src/main/java/org/folio/rest/model/filter/AccessTypeFilter.java

```diff
@@ -0,0 +1,22 @@
+package org.folio.rest.model.filter;
+
+import java.util.List;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+import org.folio.repository.RecordType;
+
+@Getter
+@Setter
+@Builder
```

Comment:
why do you need builder if you allow setters?

usually builder is used to create object and populate its properties, the object itself remains unchangeable once created
another potential use case of builder is to create objects with complex internal structure or if creation process is so complex that it's reasonable to encapsulate the process into a specific class (builder)

Reply:
Thank you for the clear explanation.